buf.go

```diff
@@ -20,7 +20,12 @@ type header struct {
 
 // bufpool provides buffers which are used for reading and writing in the tdsBuffer instances
 var bufpool = sync.Pool{
-	New: func() interface{} { return make([]byte, 1<<16) },
+	New: func() interface{} {
```

Comment:
Does this use of a pointer have any effect on your benchmark?

Reply:
There are no bench mark figures for that change in the PR, but this change is there to reduce GC of the buffers used for connections, so changing to return a pointer to the slice will not have a detrimental effect. This change was to fix the lint issue which will, happily, actually further reduce allocations.